### PR TITLE
fix: use ubi8 instead of centos7 in Dockerfile

### PR DESCRIPTION
**Pull request by chenlizhong**
### All Pull Requests:

Check all that apply:

* [ ] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

* To fix below error when build image by Dockerfile

> Could not retrieve mirrorlist http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=os&infra=container error was
14: curl#6 - “Could not resolve host: [mirrorlist.centos.org](http://mirrorlist.centos.org/); Unknown error”

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FROM centos:7
-RUN yum install -y python-devel file zip gcc libffi-devel && yum clean all
-RUN curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py && python get-pip.py && rm get-pip.py
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+RUN microdnf update -y && microdnf -y install which procps-ng git libffi-devel file xz bzip2 tar zip python3.11 python3.11-pip && microdnf clean all
+RUN python3 -m pip install --upgrade pip
```
Comment (xiangce) [inline Dockerfile:3]: `--no-cache-dir` is recommended in Dockerfile
```
python3 -m pip install --no-cache-dir --upgrade pip 
```

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 COPY . /src
 RUN pip install /src
```
Comment (xiangce) [inline Dockerfile:5]: Better update this line as the same style as the new L-3, as `python3 -m pip install --no-cache-dir /src`

File: Dockerfile (hunk 1 above)
Comment (chenlizhong) [reply Dockerfile]: Updated

File: Dockerfile (hunk 2 above)
Comment (chenlizhong) [reply Dockerfile]: Updated

**Pull request merged**